website_sale_order_type/models/website.py

```diff
@@ -0,0 +1,19 @@
+# Copyright 2023 Tecnativa - Pilar Vargas
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import models
+
+
+class Website(models.Model):
+    _inherit = "website"
+    _description = "Allowing sale_order_type to work with website_sale."
```

Comment:
You shouldn't change the model `_description`

Reply:
Good catch! This is inherited by the migration on v15, and provokes a bad problem, renaming the whole model, and sending the button in mails with such text.

Reply:
Done, thanks!